check_vmware_nsxt.py

```diff
@@ -397,19 +398,24 @@ def parse_args():
 
     args.add_argument('--version', '-V', help='Print version', action='store_true')
 
+    args.add_argument('--insecure', help='Do not verify TLS certificate. Be careful with this option, please', action='store_true', required=False)
```

Comment:
```suggestion
    args.add_argument('--insecure', help='Do not verify TLS certificate. Be careful with this option, please', action='store_false', required=False)
```

Im not a python programmer but I guess it was to be `store_false`, right? :)

After switching this to false it works with `--insecure` switch

Reply:
not my day :-)
Yes, of course